bedrock/firefox/views.py

```diff
@@ -547,6 +549,48 @@ def get_redirect_url(self, *args, **kwargs):
         return super().get_redirect_url(*args, **kwargs)
 
 
+class WhatsNewIndiaView(l10n_utils.LangFilesMixin, TemplateView):
+    def get_context_data(self, **kwargs):
+        ctx = super(WhatsNewIndiaView, self).get_context_data(**kwargs)
+
+        # add version to context for use in templates
+        version = self.kwargs.get('version') or ''
+        match = re.match(r'\d{1,2}', version)
+        ctx['version'] = version
+        ctx['num_version'] = int(match.group(0)) if match else ''
+
+        return ctx
+
+    def get_template_names(self):
```

Comment:
@pmac is there a neater way we could implement this view, without needing to duplicate all the fallback locales from the regular whatsnew view, as below?

I think the only real difference here is:

```
if locale.startswith('en-'):
    template = 'firefox/whatsnew/index-lite.in.html'
```

Reply:
I did some hacking on this and came up with:

```python
class WhatsNewIndiaView(WhatsnewView):
    def get_template_names(self):
        template = super(WhatsNewIndiaView, self).get_template_names()
        locale = l10n_utils.get_locale(self.request)
        version = self.kwargs.get('version') or ''
        channel = detect_channel(version)

        if locale.startswith('en-') and channel not in ['nightly', 'alpha', 'beta']:
            # return a list to conform with original intention
            return ['firefox/whatsnew/index-lite.in.html']

        return template
```

It seems to work and do the right things, although it would be good if @pmac could give it a sanity check.

Note: since the class extends `WhatsnewView`, I had to move it after the definition in the file for it to work.

Reply:
It would alse be good to add a unit test for this new view / template

Reply:
> It would alse be good to add a unit test for this new view / template

@alexgibson  Good idea - Do you know if there's a unit test for a similar view / template that we could use as an example?

Reply:
@sgarrity tests for the existing whatsnew view can be found here: https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/tests/test_base.py#L228

Following the same pattern for the india-specific view should work.